modules/nw-metallb-community-cr.adoc

```diff
@@ -33,9 +33,7 @@ Specify the same namespace that the MetalLB Operator uses.
 
 |`spec.communities`
 |`string`
-|Specifies a list of IP addresses for MetalLB to assign to services.
-You can specify multiple ranges in a single pool, they will all share the same settings.
-Specify each range in CIDR notation or as starting and ending IP addresses separated with a hyphen.
+|Specifies a list of BGP `CommunityAliases` to be used in BGPAdvertisements. A `CommunityAlias` consists of a pair of name (alias) and value (numeric BGP community). Link the BGPAdvertisement to a `CommunityAlias` by referring to the alias name in its `spec.communities` field.
```

Comment:
Should it be referred to as 'CommunityAlias' as if it is kind property in YAML e.g kind: Community?

Reply:
sorry @assod-rh I am a little unclear on your point. Do you want me to rephrase this ?

Reply:
sorry [@assod-rh](https://github.com/asood-rh) I am a little unclear on your point. Do you want me to rephrase this ?